conversions/decimal_to_any.py

```diff
@@ -1,6 +1,36 @@
 """Convert a positive Decimal Number to Any Other Representation"""
 
 
+ALPHABET_VALUES = {
+    "10": "A",
+    "11": "B",
+    "12": "C",
+    "13": "D",
+    "14": "E",
+    "15": "F",
+    "16": "G",
+    "17": "H",
+    "18": "I",
+    "19": "J",
+    "20": "K",
+    "21": "L",
+    "22": "M",
+    "23": "N",
+    "24": "O",
+    "25": "P",
+    "26": "Q",
+    "27": "R",
+    "28": "S",
+    "29": "T",
+    "30": "U",
+    "31": "V",
+    "32": "W",
+    "33": "X",
+    "34": "Y",
+    "35": "Z",
+}
```

Comment:
from string import ascii_uppercase

ALPHABET_VALUES = {str(ord(c) - 55): c for c in ascii_uppercase}